src/br/com/ciandt/joycelf/service/ComentarioService.java

```diff
@@ -0,0 +1,38 @@
+package br.com.ciandt.joycelf.service;
+
+/**
+ * 
```

Comment:
Faltou inserir uma descrição da classe.
```suggestion
 * Classe de serviço para comentários
```